frontend/src/app/modules/external-booking/external-booking-page/external-booking-page.component.html

```diff
@@ -63,7 +63,7 @@ <h4>{{ member.name }}</h4>
         </div>
         <div class="choice">
             <app-external-booking-choose-meeting-page
-                *ngIf="this.router.url === '/external-booking/choose-meeting' && this.menu.user"
+                *ngIf="isChooseMeetingRoute() && menu.user"
```

Comment:
can we pass all clause to the method ?

Reply:
I think it can be redundant to put both checks into one method. Since we have all methods for url check, only url check is responsible. It will not be very good if we also send a check for the user variable there